codeHF/workflows.yml

```diff
@@ -5,25 +5,52 @@ options:
 
 workflows:
 
+# Event selection
+
+  o2-analysis-timestamp:
+    activate: no
+    options:
+      mc: "--isMC 1"
+
+  o2-analysis-event-selection:
+    activate: no
+    options:
+      mc: "--isMC 1"
+
   # Skimming
 
   o2-analysis-hf-track-index-skims-creator: &skim_creator
     activate: false
     executable: o2-analysis-hf-track-index-skims-creator
     tables: [HFSELTRACK, HFTRACKIDXP2, HFTRACKIDXP3]
 
-  o2-analysis-hf-track-index-skims-creator_v0:
+  o2-analysis-hf-track-index-skims-creator-evsel:
+    <<: *skim_creator
+    activate: false
+    dependencies: [o2-analysis-timestamp, o2-analysis-event-selection]
+    options: "--doEvSel"
+    tables: [HFSELCOL, HFSELTRACK, HFTRACKIDXP2, HFTRACKIDXP3]
+
+  o2-analysis-hf-track-index-skims-creator-v0:
     <<: *skim_creator
     activate: false
     dependencies: o2-analysis-lambdakzerobuilder
     options: "--do-LcK0Sp"
     tables: [HFSELTRACK, HFTRACKIDXCASC]
 
+  o2-analysis-hf-track-index-skims-creator-evsel-v0:
+    <<: *skim_creator
+    activate: false
+    dependencies: [o2-analysis-timestamp, o2-analysis-event-selection, o2-analysis-lambdakzerobuilder]
+    options: ["--doEvSel", "--do-LcK0Sp"]
+    tables: [HFSELCOL, HFSELTRACK, HFTRACKIDXCASC]
```

Comment:
Since the `-v0` versions can run with 2/3-prong tasks now, they should include also the related tables.

Reply:
Right, I forgot that, now it is fixed